bindings/go/main.go

```diff
@@ -20,19 +17,17 @@ import (
 )
 
 const (
-	BytesPerBlob         = C.BYTES_PER_BLOB
 	BytesPerCommitment   = C.BYTES_PER_COMMITMENT
 	BytesPerFieldElement = C.BYTES_PER_FIELD_ELEMENT
 	BytesPerProof        = C.BYTES_PER_PROOF
-	FieldElementsPerBlob = C.FIELD_ELEMENTS_PER_BLOB
 )
 
 type (
 	Bytes32       [32]byte
 	Bytes48       [48]byte
 	KZGCommitment Bytes48
 	KZGProof      Bytes48
-	Blob          [BytesPerBlob]byte
+	Blob          []byte
```

Comment:
Is this type useful now that the length is unspecified? Or should we ditch the `Blob` type and have it be an array of bytes in all the functions like we did it in C? This way the `len(blob)` pattern everywhere is a bit more clear.

Reply:
Hmm. I would prefer to keep this type for readability. There's not a strong reason to keep it though.

Reply:
Sounds good. I guess it fits more how all the other bindings are structured as well.